examples_tests/22.RaytracedAO/lumaMipMapGen.comp

```diff
@@ -0,0 +1,57 @@
+#version 430 core
```

Comment:
again put in engine builtins

Reply:
This specific shader is something temporary in my opinion. my lazy ass didn't want to do twice the shader+pipeline+descriptor setup for calculating luminance and doing the mipmaps so I kinda put them both in 1 shader 😄.
Later when we use the PoTMipGenerator this shader will be only used for luma calculation

Reply:
I'd rather you put any GLSL in the engine's builtins (or generate small pre-inclusion headers at runtime), shipping GLSL with an app/artifact is messy and bugprone.